cfme/tests/automate/test_service_dialog.py

```diff
@@ -481,3 +484,128 @@ def test_automate_methods_from_dynamic_dialog_should_run_as_per_designed(
+               (for example adding the string "also").
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Values passed through to object_walker - both element values should be passed:
+               >> ~/object_walker_reader.rb | grep dialog
+                     |    $evm.root['dialog_text_box_1'] =    (type: String)
+                     |    $evm.root['dialog_text_box_2'] = The dynamic method ran at:
+                     2019-05-30 09:42:40 +0100   (type: String)
+            5. The value from both test boxes should be passed through object_walker:
+                ~/object_walker_reader.rb -t 2019-05-30T09:43:25.558037 | grep dialog
+                     |    $evm.root['dialog_text_box_1'] = aa   (type: String)
+                     |    $evm.root['dialog_text_box_2'] = The dynamic method ran at:
+                     2019-05-30 09:42:43 +0100   (type: String)
+            6. Note that both element values are now passed through to object_walker:
+                ~/object_walker_reader.rb | grep dialog
+                     |    $evm.root['dialog_text_box_1'] = ccdd   (type: String)
+                     |    $evm.root['dialog_text_box_2'] = The dynamic method also ran at:
+                     2019-05-30 09:50:10 +0100   (type: String)Provision more than 10 VMs
+    """
+    sd, ele_label = import_dialog
+    button_grp, button = custom_button(dialog=sd)
+    request.addfinalizer(button.delete_if_exists)
+    request.addfinalizer(button_grp.delete_if_exists)
+
+    user_obj = appliance.collections.users.instantiate(name="Administrator")
+    view = navigate_to(user_obj, "Details")
+    custom_button_group = Dropdown(view, button_grp.hover)
+    dynamic_value, view = navigation_to_view(custom_button_group, button)
+
+    # Step 4
+    with LogValidator("/var/www/miq/vmdb/log/automation.log",
+                      matched_patterns=[".*dialog_text_box_1:      .*",
+                                        f".*dialog_text_box_2: {dynamic_value}.*"]
+                      ).waiting(timeout=120):
+        view.submit.click()
+
+    # Step 5
+    dynamic_value, view = navigation_to_view(custom_button_group, button)
+    with LogValidator("/var/www/miq/vmdb/log/automation.log",
+                      matched_patterns=[".*dialog_text_box_1: Hello.*",
+                                        f".*dialog_text_box_2: Also{dynamic_value}.*"]
+                      ).waiting(timeout=120):
+        view.text_box1.fill("Hello")
+        view.submit.click()
+
+    # Step 6
+    dynamic_value, view = navigation_to_view(custom_button_group, button)
+    with LogValidator("/var/www/miq/vmdb/log/automation.log",
+                      matched_patterns=[".*dialog_text_box_1: Hello.*",
+                                        f".*dialog_text_box_2: {dynamic_value}.*"]
+                      ).waiting(timeout=120):
+        view.text_box1.fill("Hello")
+        view.text_box2.fill(f"Also{view.text_box2.read()}")
```

Comment:
view.text_box2.read() is same as dynamic value .
view.text_box2.fill(f"Also{dynamic_value}")

Optional : You can just return view from navigation_to_view(custom_button_group, button) and use view.text_box2.read()